Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -7,7 +7,7 @@
 //
 
 #import "Branch.h"
-#import "BranchServerInterface.h"
+#import "BNCServerInterface.h"
```

Comment:
Do we still need to import it here now that it's already imported in the header file?